blog-posts/host-listener/host-listener.md

```diff
@@ -0,0 +1,76 @@
+---
+published: false
+title: "Ain't nobody needs HostListener"
+cover_image: ""
+description: "In Angular event handling is often implemented using the hostListener decorator, even though it might not be the best fit for the problem. Have you consider more composable approaches?"
+tags: angular, rxjs, javascript, typescript
+series:
+canonical_url:
+---
+
+Angular's `@hostListener` is well known within the community. Rather unknown are the problems this might have on runtime performance and general application architecture. In general, there are two main problems with the `hostListener` decorator.
+
+1. It's not composable
+2. It can harm performance
+3. Lacks configuration options
```

Comment:
for the list items - they should either all use "It/it's" or not use any contractions